packages/iframe-stamper/src/index.ts

```diff
@@ -149,6 +155,72 @@ export class IframeStamper {
     });
   }
 
+  /**
+   * Function to inject an export bundle into the iframe
+   * The bundle should be encrypted to the iframe's initial public key
+   * Encryption should be performed with HPKE (RFC 9180).
+   * This is used during export flows.
+   */
+  async injectKeyExportBundle(bundle: string): Promise<boolean> {
```

Comment:
Just a note: it'd be nice to have some validation at this level (synchronous) to make sure the bundle has the right length / looks like a good bundle. Doesn't have to be part of this PR because this also affects injectRecoveryBundle :)

Reply:
thanks for pointing that out! will add a reminder to self to add validation across the 3 injection functions